src/main/kotlin/com/atlassian/performance/tools/awsinfrastructure/api/jira/DataCenterFormula.kt

```diff
@@ -67,7 +68,8 @@ class DataCenterFormula private constructor(
         application: com.atlassian.performance.tools.awsinfrastructure.api.storage.ApplicationStorage,
         jiraHomeSource: JiraHomeSource,
         database: Database,
-        computer: Computer
+        computer: Computer,
+        adminPasswordPlainText: String
```

Comment:
I believe you shouldn't change existing deprecated constructors. For new fields required by the main constructor you should provide a default there.

In #116 I'm introducing new pattern of `Default` object that can be used to share defaults between deprecated constructors and builders, so that those can be defined in one place. I think you could reuse this pattern unless other JPT contributors have anything against.

Reply:
How long do we want to keep deprecated constructor? Maybe we could just remove those

Reply:
There are 2 approaches. I'm not sure which JPT is using:
 * We could remove them at any point and bump major version when releasing the code.
 * We could leave them forever, as they provide value and we could live with them. Major versions don't need to break compatibility.

Reply:
We should remove them and do a major-bump when the pain of maintaining too many deprecations is bigger than the major release cascade.We should re